doc/howtos/software-update-server.rst

```diff
@@ -42,99 +42,97 @@ from the `Ostro OS download server`_
-  `OS_VERSION` variable to an integer value.
-  If set explicitly, this number must be increased before each build
-  that generates swupd update artefacts.
+#. Optionally, in ``conf/local.conf`` or equivalent location, set the
+   `OS_VERSION` variable to an integer value.
+   If set explicitly, this number must be increased before each build
+   that generates swupd update artefacts.
 
-- Assign a list of bundle names to ``SWUPD_BUNDLES``::
+#. Assign a list of bundle names to ``SWUPD_BUNDLES``::
 
-    SWUPD_BUNDLES = "feature_one feature_two"
+     SWUPD_BUNDLES = "feature_one feature_two"
 
-- For each named bundle, setup a varflag of ``BUNDLE_CONTENTS``
-  that matches the bundle name, and initialize it with a list of
-  packages whose content must be included in the bundle::
+#. For each named bundle, setup a varflag of ``BUNDLE_CONTENTS``
+   that matches the bundle name, and initialize it with a list of
+   packages whose content must be included in the bundle::
 
-    BUNDLE_CONTENTS[feature_one] = "package_one package_three package_six"
+     BUNDLE_CONTENTS[feature_one] = "package_one package_three package_six"
 
 Creating a Custom Image and matching Software Update Repository
 ===============================================================
 
 Some of the steps listed in this section are optional and cover
-advanced configuration functionality.
-Not all the users will need them, if the default configuration of
-the Ostro OS meets their needs.
+advanced configuration functionality that's only needed
+if the default configuration of
+the Ostro OS already meets your needs.
 
 The process is exemplified by using real components and describing
-what each step looks like, when applied to such components.
-Unless stated otherwise, the changes described are intended for
-the file ``conf/local.conf``.
+what each step looks like when applied to such components.
+Unless stated otherwise, the changes described are made in
+the ``conf/local.conf`` file.
 
 
 Step 0: Review and optionally modify the content of the os-core bundle
 ----------------------------------------------------------------------
-The os-core bundle comes with a predefined set of components,
+The ``os-core`` bundle comes with a predefined set of components,
 however each device is likely to have slightly different requirements.
 In some cases it might be necessary to adjust the set of components
-included in os-core.
+included in ``os-core``.
 
 To do this, either modify or extend the ``OSTRO_IMAGE_FEATURES_REFERENCE``
 and the ``OSTRO_IMAGE_INSTALL_REFERENCE`` variables for changes
 respectively to the feature and component sets.
 
-Why modifying os-core? Because it's the only unremovable bundle and
-therefore it is well suited to deliver to the device all those tools
-that must always be available.
+We're modifying ``os-core`` because it's the only unremovable bundle and is
+therefore well suited to deliver tools
+that must always be available to your devices.
```

Comment:
I'd emphasize that for most IoT products, the actual application may be in the os-core since without that, there is no use for the product :-)